<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signin</class>
 <widget class="QDialog" name="Signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{background-color:rgb(43, 49, 56);
}
QPushButton
{
border-style:bold;
border-width:0px;
border-color:green;
background-color:rgb(0, 192, 255);
font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
color:white;
border-radius:5px;
}
QPushButton:hover
{
background-color:white;
color:rgb(0, 192, 255);
border-color:green;
border-width:10px;
border-style:bold;
}
QPushButtonpressed
{
color:rgb(255, 255, 0);
background-color:rgb(52, 168, 83);
border-width:2px;}
QLabel
{
color:rgb(0,192,255);
font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;h1&gt;&lt;u&gt;SIGN-UP&lt;/u&gt;&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Print&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(0, 192, 255);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-radius:100px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
